ml_adapter/torch_autoencoder/autoencoder.ipynb

```diff
@@ -994,6 +1020,198 @@
     "await client.ml_tool.remove(ref)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "9f95dcfc-fe9a-4370-9e96-443bc22e8ce9",
+   "metadata": {},
+   "source": [
+    "## 4. Creating the plug <a id=\"plug\"></a>\n",
+    "Tell the adapter to configure itself as a plug: this generates a number of _assets_ that will be uploaded and define the plug behaviour\n",
+    "* a `plug.json` _manifest_ file that defines the name, version, _runtime_, deploy settings, metadata ...\n",
+    "* a `requirements.txt` package dependencies file\n",
+    "* a `main.py` webscript script\n",
+    "* additional scripts we added above, like the `autoencoder.py` and the `model-weights.pt`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79eba12a-193e-42e6-84ae-641fe71ae465",
+   "metadata": {},
+   "source": [
+    "#### Intiliaze adapter for plug deployment (see how to create [adapter](#adapter))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3104e1c-d950-4f1f-b407-0b171bd5cd8d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# choose a local archive location\n",
+    "ARCHIVE_LOC = 'autoencoder-pytorch'\n",
```

Comment:
I'd replace `ARCHIVE_LOC` with `PLUG_ARCHIVE_LOC` from here on , to avoid overwriting the archive location of the webscript ... 
```suggestion
    "PLUG_ARCHIVE_LOC = 'autoencoder-pytorch-plug'\n",
```